docs/introduction/test.md

```diff
@@ -54,13 +66,12 @@ The optional `fixture` argument should be an object that contains any of the fol
 import {props, state} from 'cerebral/tags'
 import {runCompute} from 'cerebral/test'
 
-import Multiply from './Multiply'
+import multiply from './multiply'
 
 it('should multiply by the specified number', () => {
-  const multiply = Multiply(state`number`, props`number`)
   const result = runCompute(multiply, {
```

Comment:
@garth I think the example for `runAction` has to be fixed in a similar way.

Reply:
done.